doc/how-to/run_migration_job_remotely.md

```diff
@@ -3,16 +3,37 @@
 To run a migration job against a non-local environment:
 
 - Ensure nobody is deploying a change (or is going to deploy a change shortly.)
-- Run `kubectl get pod` against the namespace for the environment you wish to run the seed job in.
-- Copy the name of one of the running `hmpps-approved-premises-api` pods.
+
+- Set target namespace as a variable, updating the env as required
+  ```
+  env=preprod
+  namespace=hmpps-community-accommodation-$env
+  ```
+
+- Run against the namespace for the environment you wish to run the seed job in, e.g:
+  ```
+  kubectl get pod -n $namespace
+  ```
+  
+- Copy the name of one of the running `hmpps-approved-premises-api` pods or set it as a variable, e.g:
+  ```
+  pod={pod name}
+  ```
+
 - Run the helper script from within the container to trigger the migration job:
   ```
-  kubectl exec --stdin --tty {pod name} -- /bin/bash
+  kubectl exec -n $namespace --stdin --tty $pod -- /bin/bash
   /app/run_migration_job {job type}
   ```
   Where `job type` is a value from the `MigrationJobType` enum in the OpenAPI spec.  e.g.
   ```
-  kubectl exec --stdin --tty {pod name} -- /bin/bash
+  kubectl exec -n $namespace --stdin --tty $pod -- /bin/bash
```

Comment:
thks for being explicit about the "namespace" argument as omitting this can be a recurrent gotcha...